mpo.rga.Dayon.yml

```diff
@@ -0,0 +1,59 @@
+app-id: mpo.rga.Dayon
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+command: dayon_assistant.sh
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network
+  - --device=dri
+  - --filesystem=home:rw
```

Comment:
also that kind a broad filesystem permission for a "remote control"

Reply:
The app supports sharing of the clipboard and files between `assisted` and `assistant`.

Reply:
`dri` was indeed not necessary

Reply:
`dri` is not a filesystem permission and wasn't the subject of this comment. `dri` is for access to open GL which the toolkit might want (or not).